content/docs/_index.md

```diff
@@ -6,39 +6,37 @@ weight: 1
+Falco is a behavioral activity monitor designed to detect anomalous activity in your applications. Powered by [Sysdig](https://sysdig.com/)'s [system call capture](https://sysdig.com/blog/fascinating-world-linux-system-calls/) infrastructure, Falco lets you continuously monitor and detect container, application, host, and network activity, all in one place, from one source of data, with one set of [rules](rules).
 
 ### What kind of behaviors can Falco detect?
 
-Falco can detect and alert on any behavior that involves making Linux system calls. Falco alerts can be triggered by the use of specific system calls, their arguments, and by properties of the calling process. For example, you can easily detect things like:
+Falco can detect and alert on any behavior that involves making [Linux system calls](http://man7.org/linux/man-pages/man2/syscalls.2.html). Falco alerts can be triggered by the use of specific system calls, their arguments, and by properties of the calling process. For example, you can easily detect when:
 
 * A shell is run inside a container
 * A server process spawns a child process of an unexpected type
-* Unexpected read of a sensitive file (like `/etc/shadow`)
+* A sensitive file, like `/etc/shadow`, is unexpectedly read
 * A non-device file is written to `/dev`
 * A standard system binary (like `ls`) makes an outbound network connection
 
-## How Falco Compares to Other Security Tools like SELinux, Auditd, etc.
+## How Falco compares to other tools
 
-One of the questions we often get when we talk about Falco is “How does it compare to other tools like SELinux, AppArmor, Auditd, etc. that also have security policies?”. We wrote a [blog post](https://sysdig.com/blog/selinux-seccomp-falco-technical-discussion/) comparing Falco to other tools.
+People often ask how Falco differs from [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux), [AppArmor](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=6&ved=2ahUKEwju4J3G1fjiAhVEwZ8KHZnuDhUQFjAFegQIARAB&url=https%3A%2F%2Fwiki.ubuntu.com%2FAppArmor&usg=AOvVaw1WzylrmSdwSADN7420atY5), [Auditd](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwj4n8fO1fjiAhUQBnwKHdjUBv0QFjAAegQIABAB&url=https%3A%2F%2Flinux.die.net%2Fman%2F8%2Fauditd&usg=AOvVaw3kdma9trI4_K20-nq3KBeS), and other tools related to Linux security policy. We wrote a [blog post](https://sysdig.com/blog/selinux-seccomp-falco-technical-discussion/) on the [Sysdig blog](https://sysdig.com/blog) comparing Falco to other tools.
 
-## How you use it
+## How to use Falco
 
-Falco is deployed as a long-running daemon. You can install it as a debian/rpm package on a regular host or container host, or you can deploy it as a container.
+Falco is deployed as a long-running daemon. You can install it as a [Debian](installation#debian)/[rpm](installation#rhel) package on a regular host or container host, or you can deploy it as a [container](installation#docker).
```

Comment:
Probably worth waiting https://github.com/falcosecurity/falco-website/pull/28 to be merged in so to let users know that they can also build it from source (pointing them to that documentation page).

WDYT about mentioning it here?